packages/picker/src/Picker.ts

```diff
@@ -405,13 +410,33 @@ export class PickerBase extends SizedMixin(Focusable, { noDefaultSize: true }) {
                     : html`
                           <span hidden id="applied-label">${appliedLabel}</span>
                       `}
-                ${this.invalid
+                ${this.invalid && !this.pending
                     ? html`
                           <sp-icon-alert
                               class="validation-icon"
                           ></sp-icon-alert>
                       `
                     : nothing}
+                ${when(this.pending, () => {
+                    import(
+                        '@spectrum-web-components/progress-circle/sp-progress-circle.js'
+                    );
+                    return html`
+                        <span
```

Comment:
Why this `span` you may ask:

If you want the picker to automagically add "loading" (or any other synonym) to its accessible name when `pending` is true, you need to add either this visually hidden span, or to add a label on the `sp-progress-circle` and link it with the button through `aria-labelledby`. 

When the second option is added, the screen reader actually reads the percentage of the "progress-circle" as well ("Loading 50 Choose an option.... Country") 🤦‍♀️ .

Or we can not automagically add the "loading" part to the name and let the user handle that through the picker's label.

Reply:
@TarunAdobe in a "pending button" we obsolete the (previously) visible test in the accessibility tree for the provided "Loading" text. Note that that content is _provided_ so that i18n can happen on it.

Do you think we want to dump all of the label data in this case as well?
Should we dump all of the value data and only use `${labelData} ${loadingLabel}`? 
Can you sync with the accessibility team to get their thoughts on what a screen reader should be departing to a visitor in this context?

Reply:
`When the second option is added, the screen reader actually reads the percentage of the "progress-circle" as well ("Loading 50 Choose an option.... Country") 🤦‍♀️ .`
In our case the progress-circle would be indeterminate so the screen reader should just read "Loading Choose an Option.... Country"? Am I missing something here?

Reply:
@Westbrook asked the accessibility team for some feedback and they said it's okay. The computed accessibility name for the Picker’s rendered button element will be “_Loading Select an option… Country._”
However, "Loading" is not localized. I am not aware if there's something for i18n in SWC, or else we should let the user provide a localized pending label, as the Button does.

@TarunAdobe I had the same expectation. However, unit tests fail with:
```
Browser logs on Firefox:
          {
            role: 'buttonmenu',
            name: 'Loading 50 Select an option... Country',
            haspopup: 'true'
          }
```
when using this approach:
```
<button
  .....
  aria-labelledby="loader-label icon label applied-label"
  ....
/>

......

  <sp-progress-circle
      id="loader-label"
      label="Loading"
      size="s"
      indeterminate
      class="progress-circle"
  ></sp-progress-circle>

```
Chromium and Webkit do not have this issue.

Reply:
An external label is best for right now. While we have[ this issue](https://github.com/adobe/spectrum-web-components/issues/1975), not telling when it will be prioritized/addressed.

For the Progress Circle in Firefox... it may be that that is a "situation" (neither a bug or a feature) in Firefox specifically for the Progress Circle.
- @TarunAdobe can you help confirm the actual label delivered in that case, separate of this Picker change?
  - if it's something about Progress Circle, we can make an issue and come back to it separately
  - if it's not, it may be a timing question in that Picker could be `updateComplete` before Progress Circle is, which means a couple of `nextFrame()`s might tip the scale.

Reply:
I tested the same thing in the `progress-circle.test.ts`:
```
const el = await fixture<ProgressCircle>(html`
            <input aria-labelledby="test" />
            <sp-progress-circle
                indeterminate
                label="Loading"
                id="test"
            ></sp-progress-circle>
        `);
```

result is 
```
 🚧 Browser logs on Firefox:
      {
        role: 'document',
        name: '',
        children: [
          { role: 'textbox', name: 'Loading 50' },
          {
            role: 'progressbar',
            name: 'Loading',
            valuetext: '50',
            checked: 'mixed',
            value: '50'
          }
        ]
      }
```

One way to bypass that "50" value is by using `aria-valuetext=""` although it seems hackish.

Reply:
@TarunAdobe would you please check this ^ out? I addressed all the other comments in this PR.

Reply:
Hey @Rocss... I've confirmed this behaviour in firefox and for us right now the `hackish` way seems like a slightly better option than to have a separate `<span/>` tag. So maybe we should move forward in that direction unless you have different thoughts?

Reply:
Another option would be to completely replace the label inside the picker with the `pendingLabel`, this way it would be read by screen readers automatically like: "Pending... Country"